allo/backend/llvm.py

```diff
@@ -1,6 +1,6 @@
 # Copyright Allo authors. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-# pylint: disable=no-name-in-module, inconsistent-return-statements
+# pylint: disable=no-name-in-module, inconsistent-return-statements, too-many-function-args
```

Comment:
I suspect that your black version is different from the Allo-suggested one. Basically if you check the CI outputs, this pylint flag is unnecessary.

Reply:
https://github.com/cornell-zhang/allo/blob/main/requirements.txt#L9